### ISSUE TITLE
Rename `valid_time` variable name

### ISSUE THREAD

**Issue by veenstrajelmer** [ecmwf/cdsapi#129]
### Is your feature request related to a problem? Please describe.

The time variable is called `time` for most netcdf datasets around. Since the update to the "new CDS", the data retrieved from CDS instead has a time variable called `valid_time`. This is inconvenient in our processes, so we rename it. I understand it is only a minor edit we have to do, but I was wondering if it could be considered to rename it back to `time` anyway.

### Describe the solution you'd like

Rename the time variable to `time` instead of `valid_time`

### Describe alternatives you've considered

Our current workaround, renaming the variable ourselves.

### Organisation

Deltares

**Comment by jreniel** [comment]
Yes, this is one of the things that broke my workflow as well, and it is quite inconvenient to do this kind of compat breaking changes in the upstream. It's an unfortunate and hard to justify change.

**Comment by bartvstratum** [comment]
Same here. To make matters worse, NetCDF files downloaded via MARS still follow the old naming convention. To ensure our software works with both old and new `cdsapi` files, as well as MARS, we saw no other option but to patch the new NetCDF files to make them consistent with the old ones.

I understand that some changes were necessary (e.g., updating time units from `hours since …` to `seconds since …`), but renaming variables or even dimensions is just frustrating for downstream users.
